apps/dev-portal-web/domains/docs/components/widgets/AmountDistributionCalculator/distributionItem.module.css

```diff
@@ -0,0 +1,3 @@
+.recipient-checkbox-row {
+  height: 28px;
```

Comment:
why hard-coded height? If we change checkbox size, your layout will be shifted. Why not using padding instead?

Reply:
Changed to padding